### DISCUSSION TITLE
Interop with ImGui

### DISCUSSION THREAD

**Discussion by pzoltowski** [brackeen/glfm#39] [Q&A] [answered]
I'm interested in using GLFM together with ImGui on mobile devices (mostly iOS). Imgui already has sample that works on iOS using native apple windows/views + metal backend, however input handling is not that great. I know ImGui has also GLFW backend. I'm just wondering if GLFM has mostly 1 to 1 API to GLFW and how hard would be such surgery to use GLFM instead of GLFW for interop with imgui?

**Comment by brackeen** [comment, marked as answer]
Hi @pzoltowski 
I took a quick look at imgui_impl_glfw.cpp and I think it would be possible to implement the same in GLFM, except for the clipboard and gamepad. Character input will work better on iOS than Android, though.
